cmake/CheckAbi.cmake

```
@@ -0,0 +1,30 @@
+if(DEFINED GLIBCXX_USE_CXX11_ABI)
+  message(STATUS "_GLIBCXX_USE_CXX11_ABI=${GLIBCXX_USE_CXX11_ABI} is already defined as a cmake variable")
+else()
```

As this code is in function now, one can use `return()` rather than nested `if()`->`else()`->`endif()`
```suggestion
  message(STATUS "_GLIBCXX_USE_CXX11_ABI=${GLIBCXX_USE_CXX11_ABI} is already defined as a cmake variable")
  return()
endif()
```

---

Got it.